spectrum/xpub.py

```diff
@@ -0,0 +1,65 @@
+from spectrum import logger
+
+
+LOGGER = logger.logger(__name__)
+
+def _log(message, *args, **kwargs):
+    LOGGER.info(message, extra={'app':'elife-xpub'}, *args, **kwargs)
+
+class PageObject:
+    def __init__(self, driver):
+        self._driver = driver
+
+    def _input(self, css_selector, text, name=None):
```

Comment:
Might be best to rename the method to express more what it does (send input using a css selector).